fuzzbench/run_experiment.py

```diff
@@ -46,8 +48,15 @@ def run_experiment():
 
     while True:
         print('Current status of jobs:')
+        print('\tqueued:\t%d' % len(queue))
+        print('\tstarted:\t%d' % queue.started_job_registry.count)
+        print('\tdeferred:\t%d' % queue.deferred_job_registry.count)
+        print('\tfinished:\t%d' % queue.finished_job_registry.count)
+        print('\tscheduled:\t%d' % queue.scheduled_job_registry.count)
+        print('\tfailed:\t%d' % queue.failed_job_registry.count)
```

Comment:
Do we want to print both the stats and the list of jobs?

By "stats" i mean this part:
```
run-experiment_1  |     queued: 1
run-experiment_1  |     started:        0
run-experiment_1  |     deferred:       15
run-experiment_1  |     finished:       0
run-experiment_1  |     scheduled:      0
run-experiment_1  |     failed: 0
```

If the stats are enough, we can remove the list of jobs.

If we want to keep the list of jobs, do we still need the stats, given that we print the status of the jobs (which makes the stats redundant)?

Reply:
> Do we want to print both the stats and the list of jobs?

In the long term, I think this should configured by user for information printing. Also the time period to iteration printing.

Short termly, we use this information to understand why len(queue) is zero but there are deferred jobs "enqueued" by `queue.enqueue()` statement. The fact is, deferred jobs are not in the queue, but in deferred_jobs_registry.

My opinion is to keep both of them currently.

> If the stats are enough, we can remove the list of jobs.

I would not say stats are enough.

> If we want to keep the list of jobs, do we still need the stats, given that we print the status of the jobs (which makes the stats redundant)?

I think it depends on what information we want. For stats, we can have a glance on what's going on. Just a list of status of jobs will be hard to know the numbers.
For list, that is also important and we can know which image is building now. Further, which trial is running now.

I think the question is, "we" are developers and debuggers, and we need more information.
However, if we think from the "users", we cannot assume which level of information will be better to them. We need to provide flexibility for information printing. Like mentioned above, status or stats, 3 seconds or 30 seconds, etc.

Reply:
i'd personally eliminate redundant information, but leaving it up to you

Reply:
I agree. But I guess the meaning of "redundant" depends on who runs this experiment.

Reply:
I eliminate the `scheduled_job_registry`. If finally we use that, we can add it at that time.